src/antigravity/agent.py

```
@@ -0,0 +1,376 @@
+
+class GeminiAgent:
+    """
+    A production-grade agent wrapper for Gemini.
+    Implements the Think-Act-Reflect loop with MCP integration.
+
+    The agent supports two types of tools:
+    1. Local tools: Python functions in src/antigravity/tools/ directory
+    2. MCP tools: Tools from connected MCP servers (when MCP_ENABLED=true)
+    """
+
+    def __init__(self):
```

Ruff (0.14.8)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)